### DISCUSSION TITLE
Catalog neighborhood devices and analyze SNR

### DISCUSSION THREAD

**Discussion by hdtodd** [merbanan/rtl_433#2118] [Show and tell]
In using my RTL_SDR dongle to collect temperature data from my Acurite remote sensor, I noticed that there are a number of other devices in my neighborhood sending signals on the 433MHz band.  I wondered what else is out there and how well other thermometer readings correlated with mine.  But first I had to know what was in the neighborhood.

So I wrote a pair of programs, snr (C) and SNR.py (Python), to analyze rtl_433 JSON logs and catalog and analyze signals (specifically the signal-to-noise ratio) from devices recorded in the logs.  I was astounded at the number and variety of devices in our (rather small and isolated) neighborhood.

These programs might be useful to others.  They're available on github:
```
git clone https://github.com/hdtodd/rtl_snr
```

**Comment by zuckschwerdt** [comment]
Project link: https://github.com/hdtodd/rtl_snr

Open list of ideas:

- [ ] PR #2120 to https://github.com/merbanan/rtl_433/blob/master/docs/LINKS.md
- [ ] Add reading from the [HTTP-API](https://github.com/merbanan/rtl_433/blob/master/src/http_server.c#L20-L21) for live updates.
- [ ] Port to JS to use on our HTTP UI

**Reply by hdtodd** [reply]
Proposed language added to LINKS.md

**Comment by gdt** [comment]
Now that this is in LINKS, should this discussion be closed out?
